p2p/p2p.go

```diff
@@ -43,10 +41,8 @@ type Service struct {
 	handler *starknet.Handler
 	log     utils.SimpleLogger
 
-	dht        *dht.IpfsDHT
-	pubsub     *pubsub.PubSub
-	topics     map[string]*pubsub.Topic
-	topicsLock sync.RWMutex
```

Comment:
Why no need for topics now? Were they removed from the spec?

Reply:
We just agreed on last p2p session that we're going to remove unused code for now.